tutorials/3d/spatial_material.rst

```diff
@@ -26,6 +26,9 @@ added in the *Material Override* property of the node, it will only be used by t
 It will also override the regular material property of the node and the material property of
 the mesh.
 
+The *Material Overlay* property will render a material over the current one being used by the
+mesh. As an example this could be used to put a transparent shield effect on a mesh.
```

Comment:
```suggestion
The *Material Overlay* property will render a material **over** the current one being used by the
mesh. As an example, this can be used to put a transparent shield effect on a mesh.
```